docs/use/evm-guides/rescuse-lockdrop.md

```diff
@@ -0,0 +1,29 @@
+import lockdropaccount from "/docs/build/img/lockdropacc.png"
+
+# Regaining Access to Lockdrop Accounts
+
+Due to potential security issues, the use of lockdrop accounts was discontinued, leaving some users unable to access their funds if they hadn't withdrawn them within the provided short window. We're updating Shiden and Astar to restore access to these lockdrop accounts. The steps to reconnect your lockdrop account remain unchanged:
```

Comment:
I'd suggest removing the mention of a security risk.

It's enough to say that this is a limited support window for a deprecated feature so users can get their funds out, and that lockdrop accounts were never supposed to be used for anything except claiming their ASTR lockdrop rewards.